README.md

```diff
@@ -315,6 +315,18 @@ helm template nginx-ingress \
 
 >More info about the nginx standard installation can be found on the [official nginx ingress website](https://kubernetes.github.io/ingress-nginx/deploy/).
 
+### Create Kubernetes roles and map Azure groups
+
+One or more Active Directory groups should have already been created through [Terraform](https://github.com/teamdigitale/io-infrastructure-live). It's time to create some Kubernetes roles and map them to the groups created.
+
+* Open the *dev-azure-aad-cluster-roles.yaml* file and make sure the group names are up to date. Groups names (IDs) can be found in the Azure GUI under Azure Active Directory -> Groups -> Name of the group -> Overview
+
+* Apply the roles and role bindings
+
+```shell
+kubectl apply -f system/dev-azure-aad-cluster-roles.yaml
+```
+
 ## Deploy Azure Storage PersistentVolumeClaim (PVCs) for IO services
 
 PVCs for IO services are defined outside the helm-charts to avoid their deletion, while a chart gets removed for maintenance, or simply for human errors.
```

Comment:
I'd add that you have to ensure that pvc(s) belong to the same namespace that you specify when running helm install.

Now, I wonder, since this process 1. is error prone (you can specify a namespace during helm install that does not contains the right pvc) 2. prevents us to have more than one namespace (which is the one hardcoded into the pvc yaml), can we put the pvc into the helm chart using` helm.sh/resource-policy: "keep"` https://github.com/helm/helm/issues/6261#issuecomment-523472128 ?
this would let us avoid hardcoding namespaces into resources.

Reply:
Including the PVC in the helm-chart is not a good practice, since the PVC would be deleted every time the helm-chart gets deleted for any reason. It must remain out of the helm-chart

Reply:
ok but there's a way to tell tiller to not delete a resource: https://helm.sh/docs/howto/charts_tips_and_tricks/#tell-helm-not-to-uninstall-a-resource